examples/tensorrt_llm/configs/llm_api_config_disagg_router.yaml

```diff
@@ -38,4 +38,4 @@ kv_cache_config:
 pytorch_backend_config:
   enable_overlap_scheduler: false
   use_cuda_graph: false
-  enable_iter_perf_stats: true
```

Comment:
can you unstage this file's changes so it's clearer that these files are actually unchanged since an older commit?